### PR TITLE
Add gimbal v2 status and mission items

### PR DESCRIPTION
**Pull request by julianoes**
This enables:

- attitude feedback for the gimbal v2 protocol
- mission items using gimbal v2 protocol

### CODE REVIEW

File: src/plugins/mission/mission_impl.cpp [hunk 1]
```diff
@@ -85,6 +104,35 @@ void MissionImpl::process_mission_item_reached(const mavlink_message_t& message)
     report_progress();
 }
 
+void MissionImpl::process_gimbal_manager_information(const mavlink_message_t& message)
+{
+    // mavlink_gimbal_manager_information_t gimbal_manager_information;
+    // mavlink_msg_gimbal_manager_information_decode(&message, &gimbal_manager_information);
```
Comment (JonasVautherin) [inline src/plugins/mission/mission_impl.cpp:110]:
```suggestion
```

**Review by JonasVautherin — CHANGES_REQUESTED** [review]

File: src/plugins/mission/mission_impl.cpp (hunk 1 above)
Comment (julianoes) [reply src/plugins/mission/mission_impl.cpp:110]: Yer, you're right.

**Comment by julianoes** [comment]
@JonasVautherin thanks for the review. I'm just waiting for https://github.com/mavlink/MAVSDK-Proto/pull/184 before pushing here again.

**Review by JonasVautherin — APPROVED** [review]
LGTM

**Pull request merged**